Missing parameter --zookeeper to read Kafka records script

fbeltrao opened the pull request:
The script kafka-console-consumer.sh requires the --zookeeper parameter which can be resolved from $KAFKAZKHOSTS

azurecla commented:
Hi __@jaymathe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaymathe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

fbeltrao committed:
Missing parameter --zookeeper to read Kafka records script

The script kafka-console-consumer.sh requires the --zookeeper parameter which can be resolved from $KAFKAZKHOSTS
